lib/relianceHMO/Main.js

```diff
@@ -0,0 +1,79 @@
+const request = require('./common/request');
+
+exports.Plans = async data => {
+    try {
+        return await request.trigger({
+            method: 'GET',
+            path: '/relianceHMO/plans',
+            credentials: { ...data },
+            payload: data.payload,
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+exports.EnrolleesGet = async data => {
+    try {
+        return await request.trigger({
+            method: 'GET',
+            path: '/relianceHMO/enrollees',
+            credentials: { ...data },
+            payload: data.payload,
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+exports.EnrolleesPost = async data => {
```

Comment:
You could rename the functions to something more descriptive like:
Register -> instead of EnrolleesPost and 
Enrollees -> instead of EnrolleesGet

Reply:
done.